### PR TITLE
All doctors and speciality

### PR DESCRIPTION
**Pull request by francoisedumas**
Included first_name, last_name and speciality in the query

### CODE REVIEW

File: all_doctors.sql [hunk 1]
```diff
@@ -1 +1 @@
-SELECT last_name, first_name FROM doctors ORDER BY last_name ASC;
+SELECT last_name, speciality, first_name FROM doctors ORDER BY last_name ASC;
```
Comment (FrancoisLaposte) [inline all_doctors.sql:1]: Cool this query I forget to add the speciality!
Could you do a proper indentation?

**Review by FrancoisLaposte — COMMENTED** [review]
Cool, just 1 fix!

File: all_doctors.sql (hunk 1 above)
Comment (francoisedumas) [reply all_doctors.sql]: Done!

**Review by FrancoisLaposte — APPROVED** [review]
Amazing job!!

**Pull request merged**